lang/test/org/partiql/lang/eval/EvaluatingCompilerOffsetTests.kt

```diff
@@ -110,7 +110,10 @@ class EvaluatingCompilerOffsetTests : EvaluatorTestBase() {
     @ParameterizedTest
     @ArgumentsSource(ArgsProviderValid::class)
     fun validTests(tc: EvaluatorTestCase) = runEvaluatorTestCase(
-        tc.copy(excludeLegacySerializerAssertions = true),
+        tc.copy(
+            excludeLegacySerializerAssertions = true,
+            targetPipeline = EvaluatorTestTarget.COMPILER_PIPELINE, // planner & phys. alg. have no support for OFFSET (yet)
```

Comment:
From your comment in the PR that set the test exclusions, #586, you left a [comment](https://github.com/partiql/partiql-lang-kotlin/pull/586#discussion_r867138242) saying that this comment was wrong. Perhaps there was some missed commit in the merge?

(The original version was just `tc.copy(excludeLegacySerializerAssertions = true)` and the exclusions were set by the individual `EvaluatorTestCase`s.)

Reply:
Oh, yeah, it must have returned after a merge.  Will fix.  Thanks.